pyuvdata/uvbeam/tests/test_beamfits.py

```diff
@@ -774,3 +774,35 @@ def test_multi_files(cst_efield_2freq, tmp_path):
 
     beam1.history = beam_full.history
     assert beam1 == beam_full
+
+
+@pytest.fixture(scope="module")
+def twofreq(cst_efield_2freq_main, tmp_path_factory):
+    testfile = str(tmp_path_factory.mktemp("beams") / "twofreq.fits")
+    cst_efield_2freq_main.write_beamfits(testfile)
+    return testfile
+
+
+def test_partial_read_freq(twofreq):
```

Comment:
Can we also test healpix beam files? I think that the frequency selection will just work and I also want to make sure there aren't places where the az/za ranges come in when they shouldn't.